daffodil-lib/src/main/scala/org/apache/daffodil/api/DaffodilTunables.scala

```diff
@@ -72,6 +72,20 @@ case class DaffodilTunables(
   //
   val requireBitOrderProperty: Boolean = false,
   //
+  // If true, require that the choiceDispatchKeyKind property is specified. If false, use a
```

Comment:
Use scaladoc syntax to document these members.

Reply:
I appear to be consistent with the existing convention for these files. Should I update all the comments?

Reply:
They were done wrong by prior author (which could be me.... but it got past code review at that time). 

Might as well fix them all.

Reply:
What is the appropriate usage of scaladoc here?

Since these are all class parameters, it would seem that the correct style would be to move all of the documentation to above the class with a bunch of @param annotations, and not have any comment where they are declared. Is this desired?

Reply:
Per Mike's email to me, these are class parameters, not normal fields. Since these are not part of an external API, it is not worth the added maintance effort to maintain proper scala docs, so we are leaving them as is.